src/contour/ui.template/Terminal.qml.in

```diff
@@ -28,6 +28,31 @@ ContourTerminal
         focus : false
     }
 
+
+
+    Rectangle {
+        Timer {
+            id: sizeWidgetTimer
+            interval: 1000;
+            running: false;
+            onTriggered: sizeWidget.visible = false
+        }
+        id : sizeWidget
+        anchors.centerIn: parent
+        width:  80
+        height: 30
+        color: "white"
+        visible : false
+        focus : false
+        Text {
+            id : sizeWidgetText
+            anchors.centerIn: parent
+            font.pointSize: 12
```

Comment:
I think the font size for this text should be the same as for the terminal's used font.

Reply:
Updated, now we use terminal font size